src/content/docs/apm/agents/nodejs-agent/getting-started/monitor-nodejs-app.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Monitor your Node.js application
+tags:
+  - Agents
+  - Nodejs agent
+  - Getting started
+metaDescription: 'For system requirements, installation and configuration procedures, and troubleshooting tips for New Relic''s Node.js agent, start here.'
+redirects:
+  - /docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs
+  - /docs/nodejs/new-relic-for-nodejs
+  - /docs/agents/nodejs-agent/getting-started/new-relic-nodejs
+  - /docs/agents/node-agent
+  - /docs/agents/nodejs-agent/table-of-contents
+  - /docs/apm/agents/nodejs-agent
+  - /docs/agents/nodejs-agent
+  - /docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs
+---
+
+Want to monitor your Node.js application with New Relic? You're in the right place. Use this guide to find the best install path to instrument your app with the Node.js agent. 
+
+Still need more information? [Introduction to APM](/docs/apm/new-relic-apm/getting-started/introduction-apm/) breaks down how application monitoring works and overviews how to use your data. [Introduction to New Relic for Node.js](/docs/apm/agents/nodejs-agent/getting-started/introduction-newrelic-nodejs) walks you through the install process, from beginning to end.
+
+## Choose your install method
+
+We have some options for our install, depending on how your app is deployed.  
+
+<CollapserGroup>
+  <Collapser
+        id="package-manager"
+        title="Install with a package manager"
```

Comment:
I know this is unrelated as you're just linking it but we're trying to get customers to load our agent with `-r newrelic` instead of editing user code. You can see it in our [code repo](https://github.com/newrelic/node-newrelic#installation)

```sh
 $ node -r newrelic your-program.js
```